processor/src/test/java/org/mapstruct/ap/test/builtin/BuiltInTest.java

```diff
@@ -354,22 +362,88 @@ public void shouldApplyBuiltInOnZonedDateTimeToCalendar() throws ParseException
     }
 
     @Test
-    @IssueKey( "277" )
```

Comment:
LOL, a shame really that we added a test for not supporting this back then rather than implementing the conversion :)